### ISSUE TITLE
[Mod Backport/Concept] UniversalChatl; A means for ingame communication between languages

### ISSUE THREAD

**Issue by Shibva** [GTNewHorizons/GT-New-Horizons-Modpack#10885]
### Your GTNH Discord Username

Shibva_

### Your Pack Version

Irrelivent but 2.1.2.3

### Your Proposal

Backporting and reworking a mod Called UniversalChat in newer versions (quilt/frabic mod) to work in GTNH or a mod that could be made to work like it to actively translate chat messages ingame

### Your Goal

To bridge a gap that is the Language Barrier

### Your Vision

Various groups in GTNH discord have their own language, yet from what I can deduce, most of it is English Speaking, and the official GTNH servers are such. This can impede others who may not be able to communicate in the English Language (regardless if they can't speak it or struggle to) With a mod or concept like UniversalChat, It could allow for such groups to play on the servers without the Language Barrier creating communication issues. This mod/concept will allow for such problems to be mitigated to a degree (while some are difficult to translate, its a translation that could be interpreted regardless)

TLDR: a mod, whether a backport of an existing one or a new one based on the concept of UniverstalChat to aid in communication between people between different languages

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by boubou19** [comment]
use the proper template for mod suggestion

**Comment by Shibva** [comment]
sorry, tbh, Idk what to put this under; thanks for clarification
